api/nakadi-event-bus-api.yaml

```diff
@@ -1151,6 +1248,38 @@ paths:
 # ################################### #
 
 definitions:
+  Storage:
+      type: object
+      description: |
+        A storage backend.
+      properties:
+        id:
+          description: The ID of the storage backend.
+          type: string
+          maxLength: 36
+        storage_type:
+          description: the type of storage
+          type: string
+          enum:
```

Comment:
Using enum in API makes it impossible to change in compatible way. The reason is following: client writes the code that calls GET on this resource and he expects to get enum here with exactly one possible value. If we will extend enum in the future - that will break a client. So it's better to make it a string and describe possible values.
Like here for example: https://github.com/zalando/nakadi/blob/master/api/nakadi-event-bus-api.yaml#L1773

The example of this problem you can see in BatchItemResponse definition that has several enum properties that we are not able to change any more (and we already wanted to do that several times)